packages/features/portfolio/src/containers/portfolio-holdings/use-detailed-holdings.tsx

```diff
@@ -49,6 +51,11 @@ export function useDetailedHoldings() {
           t.nonNoteEarnings = t.nonNoteEarnings.add(
             statement.totalProfitAndLoss.toFiat(baseCurrency)
           );
+
+          // TODO: need to update this return value
```

Comment:
@matthew-garrett We would need to return something like what we do in `toolTipData` below so that the total would could show all of the total incentives:

```
                    ...totalIncentiveEarnings.map((i) => ({
                      underlying: i.toDisplayStringWithSymbol(),
                      baseCurrency: i
                        .toFiat(baseCurrency)
                        .toDisplayStringWithSymbol(),
                    })),
                  ],
```